java/CustomMetrics/README.md

```diff
@@ -0,0 +1,37 @@
+# Custom Metrics
+
+* Flink version: 1.15.2
```

Comment:
can we make it 1.15.4?

Reply:
bumped flink version

Reply:
Not a change request: I wonder if we should write `1.15` not to confuse the reader, because MSF runs on 1.15.2  but we use 1.15.4 for additional dependencies.
LGTM for this anyhow